content/applications/inventory_and_mrp/inventory/shipping_receiving/reservation_methods.rst

```diff
@@ -101,3 +101,8 @@ Operations Types`, and then clicking on a specific operations type.
 .. image:: reservation_methods/reservation-methods-operations-type-menu.png
    :align: center
    :alt: Operations Types highlighted from the Configurations submenu in the Inventory app.
+
```

Comment:
If the `:show-toc:` metadata is not being used, then the sub-docs will not be linked at the bottom of `reservation_methods.rst`. In this case, you can link to the sub-docs with a seealso :)
```suggestion
.. seealso::
   :doc:`reservation_methods/at_confirmation`

```